financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsAccountsRepository.kt

```diff
@@ -151,13 +151,13 @@ private class FinancialConnectionsAccountsRepositoryImpl(
     companion object {
         internal const val PARAM_SELECTED_ACCOUNTS: String = "selected_accounts"
 
-        internal const val accountsSessionUrl: String =
+        internal val accountsSessionUrl: String =
```

Comment:
What is the point of this change?

Reply:
Since it's created from a non const val, it also can't be const. We need the base API non const so we can change from stripe.com -> localhost